perun-base/src/main/resources/perun-roles.yml

```diff
@@ -2965,18 +3040,25 @@ perun_policies:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Member
+      - MFA: User
 
   sponsored-removeSponsor_Member_User_policy:
     policy_roles:
       - SPONSORSHIP: Member
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
```

Comment:
again, I'd go with Member, User and Vo for `extendExpirationForSponsoredMember_Member_User_policy`, `sponsored-removeSponsor_Member_User_policy`, `sponsor-removeSponsor_Member_User_policy`

Reply:
Yes, this makes more sense than a global check.